packages/botfuel-dialog/src/messages/postback-message.js

```diff
@@ -23,20 +27,19 @@ const Message = require('./message');
 class PostbackMessage extends Message {
   /**
    * @constructor
-   * @param {String} dialog - the postback dialog name
-   * @param {Object[]} entities - the dialog entities
+   * @param {DialogData} dialogData - the postback dialog information
    * @param {Object} [options] - the message options
    */
-  constructor(dialog, entities, options) {
-    super('postback', 'user', { dialog, entities }, options);
+  constructor(dialogData: DialogData, options?: Object) {
+    super('postback', 'user', { dialogData }, options);
```

Comment:
Why adding this extra property `dialogData` instead of juste passing the object as `message.value` ?

As we discussed last time this is simpler to have something like:

```javascript
super('postback', 'user', dialogData, options);
```